pkg/util/containerd/namespaces.go

```diff
@@ -23,7 +23,31 @@ func NamespacesToWatch(ctx context.Context, containerdClient ContainerdItf) ([]s
 		return namespaces, nil
```

Comment:
shouldn't we exclude `containerd_exclude_namespaces` before returning here?

Reply:
It's a bit silly to set a namespace in `containerd_namespaces` just to exclude it in `containerd_exclude_namespaces`, since there's no wildcards or anything. In any case, it's documented:

```
When containerd_namespaces is set to [], containerd_exclude_namespaces
allows the exclusion of containers from specific namespaces.
```